api/client/samples/similar_regions/similar_region.py

```diff
@@ -350,7 +350,7 @@ def _load_property_from_file(self, prop_name, path_prefix):
         # will return array of long-term averages for each time interval within a year, even for static
         result = np.zeros(self.t_int_per_year)
         try:
-            raw_data = pd.read_csv(full_path, parse_dates=[0,1], infer_datetime_format=True)
+            raw_data = pd.read_csv(full_path, parse_dates=[0,1], infer_datetime_format=True).dropna()
         except Exception as e:
             self._logger.error("Can not get data from file {}: {}".format(full_path, str(e)))
             return None, [] # empty list signals no data
```

Comment:
As always with error handling this depends on preference/style, I think. We have two different ways of loading data - from Gro/cache and a file. Gro/cache variant of loading already returns empty list of valid regions if it does not get data (and can't just throw unhandled Exception since we load incrementally and might get data for other regions), so I just force the same processing by caller by returning empty valid region list here as well.